translations/messages.en.yml

```diff
@@ -12,8 +12,8 @@ page.navigation.route.sp_demo: Demo SP
 
 page.footer.terms: Terms and Conditions
 
-page.index.title: Welcome to the Azure MFA GSSP
-page.index.body.intro: TODO Introduction
+page.index.title: Welcome to the Azure MFA
+page.index.body.intro: Dit is de langsingpagina voor de Azure MFA plugin in Openconext-stepup
```

Comment:
This is the `en` translation file. I suggest translating this to English

`This is the landing page for the Azure MFA plugin in OpenConext-Stepup`

Reply:
I seem to have swapped the en and english translations.